### PR TITLE
Add Package Types page

### PR DESCRIPTION
**Pull request by KevinJump**
Add a page to describe Differences between Nuget and Zip packages: issue #2066 

_(this is my first pr for the docs, so probably needs work)._

### CODE REVIEW

File: Extending/Packages/package-types.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Document Types
+* Templates
+* Stylesheets
+* Macros
+* Languages
+* Dictionary Items
+* Data Types
+
+:::note
+Currently you cannot include media items in a Package Zip file. You will need a custom [package action](package-actions.md) to install media as part of a package.
```
Comment (sofietoft) [inline Extending/Packages/package-types.md]: When Jespers PR #2064 gets merged, this link needs to be updated, right? 
 
```
[package action](Package-Actions)
```

Comment (sofietoft) [reply Extending/Packages/package-types.md]: And the same goes for the link in line 32 - I can see that's the same at least!

File: Extending/Packages/package-types.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Package Zip files have been developed for Umbraco and have a number of advantages.
+
+Packages uploaded to our.umbraco.com are listed in the package directory and are accessible directly through the Umbraco backoffice
+
+Package zip files can add items to Umbraco and you can change Umbraco config and settings via package actions.
+
+
+## NuGet Packages
+
+A NuGet package is a standard way of delivering Compiled Code and configuration to a .net project. NuGet packages contain dlls, and files required for the solution. for more information on NuGet packages see Microsoft's [An introduction to NuGet](https://docs.microsoft.com/en-us/nuget/what-is-nuget) documentation.
```
Comment (sofietoft) [inline Extending/Packages/package-types.md]:
```suggestion
A NuGet package is a standard way of delivering Compiled Code and configuration to a .NET project. NuGet packages contain dlls, and files required for the solution. For more information on NuGet packages see Microsoft's [An introduction to NuGet](https://docs.microsoft.com/en-us/nuget/what-is-nuget) documentation.
```

File: Extending/Packages/package-types.md (hunk 1 above)
Comment (KevinJump) [reply Extending/Packages/package-types.md]: yep I've updated them now, hopefully that's the right page.

File: Extending/Packages/package-types.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Packages uploaded to our.umbraco.com are listed in the package directory and are accessible directly through the Umbraco backoffice
+
+Package zip files can add items to Umbraco and you can change Umbraco config and settings via package actions.
+
+
+## NuGet Packages
+
+A NuGet package is a standard way of delivering Compiled Code and configuration to a .NET project. NuGet packages contain dlls, and files required for the solution. For more information on NuGet packages see Microsoft's [An introduction to NuGet](https://docs.microsoft.com/en-us/nuget/what-is-nuget) documentation.
+
+NuGet packages can be installed using NuGet.exe via the a command line or through the [Package Manager Console](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-powershell) within Visual Studio using a command similar to the one below:
```
Comment (jmayntzhusen) [inline Extending/Packages/package-types.md]:
```suggestion
NuGet packages can be installed using NuGet.exe via the command line or through the [Package Manager Console](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-powershell) within Visual Studio using a command similar to the one below: 
```

File: Extending/Packages/package-types.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+A NuGet package is a standard way of delivering Compiled Code and configuration to a .NET project. NuGet packages contain dlls, and files required for the solution. For more information on NuGet packages see Microsoft's [An introduction to NuGet](https://docs.microsoft.com/en-us/nuget/what-is-nuget) documentation.
+
+NuGet packages can be installed using NuGet.exe via the a command line or through the [Package Manager Console](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-powershell) within Visual Studio using a command similar to the one below: 
+
+```
+PM> Install-Package MyPackage -version 1.2
+```
+
+NuGet packages can include any solutions files, and can be configured to run powershell scripts after installation,
```
Comment (jmayntzhusen) [inline Extending/Packages/package-types.md]:
```suggestion
NuGet packages can include any solutions files, and can be configured to run powershell scripts after installation. 
```

**Comment by jmayntzhusen** [comment]
Awesome Kevin! 🙂 And congrats on your first Documentation PR 😉🎉

**Pull request merged**